### ISSUE TITLE
Engintron 1.10 on CentOS 7.6 with MariaDB 10.3 > database status error

### ISSUE THREAD

**Issue by mjpromotions** [engintron/engintron#1072]
Hi,
On CentOS 7.6 MariaDB is started via systemd service instead of the systemV scripts.
When checking in Engintron > database status, it says > 
`Database Status:
MariaDB is not running[FAILED]`
This indicates the following command is used >
`service mysql status`

Can anyone confirm this bug ?

**Comment by fevangelou** [comment]
Unless the alias "service" is removed, I don't see a reason why this shouldn't work. In any case, if you run `systemctl status mariadb` what do you get?

**Comment by mjpromotions** [comment]
`[root@server ~]# systemctl status mariadb
● mariadb.service - MariaDB 10.3.16 database server
   Loaded: loaded (/usr/lib/systemd/system/mariadb.service; disabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/mariadb.service.d
           └─migrated-from-my.cnf-settings.conf, protecthome.conf
   Active: active (running) since Wed 2019-06-19 07:05:33 EDT; 4h 0min ago
     Docs: man:mysqld(8)
           https://mariadb.com/kb/en/library/systemd/
  Process: 10252 ExecStartPost=/bin/sh -c systemctl unset-environment _WSREP_START_POSITION (code=exited, status=0/SUCCESS)
  Process: 10180 ExecStartPre=/bin/sh -c [ ! -e /usr/bin/galera_recovery ] && VAR= ||   VAR=`/usr/bin/galera_recovery`; [ $? -eq 0 ]   && systemctl set-environment _WSREP_START_POSITION=$VAR || exit 1 (code=exited, status=0/SUCCESS)
  Process: 10151 ExecStartPre=/bin/sh -c systemctl unset-environment _WSREP_START_POSITION (code=exited, status=0/SUCCESS)
 Main PID: 10216 (mysqld)
   Status: "Taking your SQL requests now..."
   CGroup: /system.slice/mariadb.service
           └─10216 /usr/sbin/mysqld

Jun 19 07:05:28 server-ip.net systemd[1]: Starting MariaDB 10.3.16 database server...
Jun 19 07:05:28 server-ip.net mysqld[10216]: 2019-06-19  7:05:28 0 [Note] /usr/sbin/mysqld (mysqld 10.3.16-MariaDB) starting as process 10216 ...
Jun 19 07:05:33 server-ip.net systemd[1]: Started MariaDB 10.3.16 database server.`

**Comment by fevangelou** [comment]
Looks like it's disabled, so if for some reason it stopped, you need to manually start it.

Just do:
```
systemctl enable mariadb
```
...and I'm guessing `service X...` will work again.

**Comment by mjpromotions** [comment]
Hi, `service mysql status` still not working. 


As I understand form centos forums > 

> Because the service was started via systemd service instead of the systemV scripts.

You should avoid using the 'service' command on CentOS7

>

**Comment by mjpromotions** [comment]
Also reading here
https://linuxize.com/post/install-mariadb-on-centos-7/

systemctl is the default command on CentOS 7

**Comment by fevangelou** [comment]
"service" remains though as an alias to systemctl. In your case, the DB seems to have been down.

**Comment by mjpromotions** [comment]
Sorry, it doesnt make sense to me. When I do >
`systemctl status mysql`
it says
`● mariadb.service - MariaDB 10.3.16 database server
   Loaded: loaded (/usr/lib/systemd/system/mariadb.service; enabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/mariadb.service.d
           └─migrated-from-my.cnf-settings.conf, protecthome.conf
   Active: active (running) since Wed 2019-06-19 11:19:13 EDT; 38min ago
     Docs: man:mysqld(8)
           https://mariadb.com/kb/en/library/systemd/
 Main PID: 7496 (mysqld)
   Status: "Taking your SQL requests now..."
   CGroup: /system.slice/mariadb.service
           └─7496 /usr/sbin/mysqld

Jun 19 11:18:56 server-ip.net systemd[1]: Starting MariaDB 10.3.16 database server...
Jun 19 11:18:59 server-ip.net mysqld[7496]: 2019-06-19 11:18:59 0 [Note] /usr/sbin/mysqld (mysqld 10.3.16-MariaDB) starting as process 7496 ...
Jun 19 11:19:13 server-ip.net systemd[1]: Started MariaDB 10.3.16 database server.`

This would indicate mariadb is running for 38 minutes right ?

And then I do >
`service mysql status`
and it says 
`MariaDB is not running                                     [FAILED]`

As explained to me on CentOS forum it suggests the above command wont work, while mariadb IS running.
This is the way CentOS 7 works I am told ?

**Comment by mjpromotions** [comment]
Hi,
This thread might clarify stuff a bit more 
https://www.centos.org/forums/viewtopic.php?f=48&t=70840&p=297903

**Comment by mjpromotions** [comment]
Hi, can you tell me why the issue is closed ? As I get it it really is CentOS 7 that has moved to systemctl ?

Thanks

**Comment by fevangelou** [comment]
There is no issue here. I've told you 3-4 times already that "service" is an alias in CentOS 7 (as it's on Ubuntu and others). The fact that MariaDB was disabled on your server indicates that someone probably messed up with it in the first place.

**Comment by mjpromotions** [comment]
Can I send you a PM ?

**Comment by sioulisn** [comment]
Yesterdat I changed my tables in all databases from MyISAM to InnoDB.
Today after I restarted my server the same issue occured without touching anything.
Mariadb was active but in Engintron i was getting this:
`Database Status: MariaDB is not running[FAILED]`

So I just used the command in SSH:
`systemctl enable mariadb`

and now is ok, just I haven't restarted my server again to check if this will happen again. 
Is not something related to Engintron just very lucky we have the Engintron panel and can check if everything is ok!

thank you Fotis
